redsys/__init__.py

```diff
@@ -133,6 +134,16 @@ def redsys_generate_request(self, params):
                 params.get('DS_MERCHANT_CONSUMERLANGUAGE'), '001'),
             }
 
+        # This field is only needed when we use some special, payment, if we
```

Comment:
DS_MERCHANT_PAYMETHODS is a optional parameter.
I think it helps to copy documentation to see options:

```
# BIZUM
# DS_MERCHANT_PAYMETHODS: z
# DS_MERCHANT_TRANSACTIONTYPE: 0,7,8
# PAYPAL: 
# DS_MERCHANT_PAYMETHODS: P
# DS_MERCHANT_TRANSACTIONTYPE: 0,1,15
# Transferencia:
# DS_MERCHANT_PAYMETHODS: R
# DS_MERCHANT_TRANSACTIONTYPE: 0,15
# Con Tarjeta:
# DS_MERCHANT_PAYMETHODS: C
# DS_MERCHANT_TRANSACTIONTYPE: TODAS	
# GooglePay y ApplePay:
# DS_MERCHANT_PAYMETHODS: xpay
# DS_MERCHANT_TRANSACTIONTYPE: 0,7
```